src/components/AppLayout/Header/components/NetworkSelector.tsx

```diff
@@ -92,19 +91,16 @@ const NetworkSelector = ({ open, toggle, networks, clickAway }: NetworkSelectorP
   const networkName = getNetworkName().toLowerCase()
 
   const onNetworkSwitch = useCallback(
-    (safeUrl: string, networkId: ETHEREUM_NETWORK) => {
-      // FIXME remove navigation when L2-UX completes
-      // This was added in order to switch network using navigation on prod
-      // but be able to check chain swapping on dev environments and PRs
-      if (APP_ENV === 'production') {
```

Comment:
I think that `(APP_ENV === 'production'` check is needed right? its our network switch feature flag

Reply:
I'm pretty sure this isn't necessary anymore as this branch is based off the `l2-ux` branch. Can you confirm @dasanra?

Reply:
I think it's fine to remove it now, as we are planning to redirect to the main domain and we expect this feature to be released once L2-UX is completed